docs/source/en/api/schedulers/overview.mdx

```diff
@@ -80,7 +80,3 @@ The base class [`SchedulerMixin`] implements low level utilities used by multipl
 The class [`SchedulerOutput`] contains the outputs from any schedulers `step(...)` call.
 
 [[autodoc]] schedulers.scheduling_utils.SchedulerOutput
-
-### KarrasDiffusionSchedulers
-
-[[autodoc]] schedulers.scheduling_utils.KarrasDiffusionSchedulers
```

Comment:
The doc builder doesn't know how to autodoc Enums. I think it's ok to remove

Reply:
should we add it manually?

Reply:
Yeah that's what I'm thinking : https://github.com/huggingface/diffusers/pull/2346#issuecomment-1430272898